lib/iris/fileformats/um/_fast_load.py

```diff
@@ -40,19 +40,67 @@
 import threading
 import os.path
 
+import numpy as np
+
 # Be minimal about what we import from iris, to avoid circular imports.
 # Below, other parts of iris.fileformats are accessed via deferred imports.
 import iris
 from iris.coords import DimCoord
 from iris.cube import CubeList
 from iris.exceptions import TranslationError
-
+from iris.fileformats.um._fast_load_structured_fields import \
+    BasicFieldCollation, group_structured_fields
```

Comment:
'group_structured_fields' was originally a dynamic import.
Don't think that was necessary (seems to work).